apps/site/assets/ts/schedule/components/schedule-finder/UpcomingDepartures.tsx

```diff
@@ -139,7 +139,30 @@ export const UpcomingDepartures = ({
   if (journeys !== null && hasPredictions(journeys)) {
     return (
       <>
-        <h3>Upcoming Departures</h3>
+        <div className="schedule-table__upcoming-departures-header">
+          <h3>Upcoming Departures</h3>
+          &nbsp;
+          <span className="schedule-table__live-clock" aria-hidden>
+            <span className="icon-realtime animate">
```

Comment:
This extra `span.icon-realtime.animate` wrapper is redundant as far as I can tell.

Reply:
I will check